README.md

```diff
@@ -16,6 +16,8 @@ If on go 1.12 then ensure the project either resides outside of your `GOPATH` or
 | GRACEFUL_SHUTDOWN_TIMEOUT    | GracefulShutdownTimeout    | Time to wait during graceful shutdown                                                  | 5 seconds               |
 | HEALTHCHECK_INTERVAL         | HealthCheckInterval        | Interval between health checks                                                         | 30 seconds              |
 | HEALTHCHECK_CRITICAL_TIMEOUT | HealthCheckCriticalTimeout | Amount of time to pass since last healthy health check to be deemed a critical failure | 90 seconds              |
+| OTEL_EXPORTER_OTLP_ENDPOINT         | localhost:4317                         | Endpoint for OpenTelemetry service
+| OTEL_SERVICE_NAME                   | dp-frontend-cookie-controller          | Label of service for OpenTelemetry service
```

Comment:
Could the | characters be brought in line with the rows above please?

Reply:
Aligned the rows. I was also using the default values in the wrong place so have corrected that too.